modules/ossm-extensions-wasm-support.adoc

```diff
@@ -0,0 +1,33 @@
+[id="ossm-extensions-support_{context}"]
+= Enabling WebAssembly extension support
+
+Support for WebAssembly extensions to {ProductName} is currently in Tech Preview, so it must be explicitly enabled for your `ServiceMeshControlPlane`. 
```

Comment:
Maybe add the link for technology preview:

Support for WebAssembly extensions to {ProductName} is currently in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview], so it must be explicitly enabled for your `ServiceMeshControlPlane`.